proxy/proxy_test.go

```diff
@@ -612,6 +613,82 @@ var _ = Describe("Proxy", func() {
 		})
 	})
 
+	FDescribe("HTTP Rewrite", func() {
```

Comment:
@zachgersh this has been merged with a focus! 

I didn't expect to be merged without agreeing the right interface, sorry for that.

Shall we do a PR asap to fix thix

Reply:
Fixed in e2b2187dc9af8a233005d5edecf168900e9b1e78 :+1:

Reply:
@keymon - yeah we caught the focus and some others (no big deal). The code as implemented worked (so we were happy to not delay a merge). A fast follow PR to fix the yaml structure that @shalako wants would be great (and will be merged quickly as well).